src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -10,8 +10,8 @@
  */
 public class GuiSettings implements Serializable {
 
-    private static final double DEFAULT_HEIGHT = 600;
-    private static final double DEFAULT_WIDTH = 740;
+    private static final double DEFAULT_HEIGHT = 900;
```

Comment:
Any reason for this specific resolution? Perhaps a generic 1280x720 or 1920x1080 would be consistent among different computers.

Reply:
Current resolution updated to 1280x720 for better compatibility